deployment/terraform/container_service.tf

```diff
@@ -0,0 +1,201 @@
+locals {
+  app_image = "${module.ecr-repository.repository_url}:${var.image_tag}"
+}
+
+#
+# Security Group Resources
+#
+resource "aws_security_group" "alb" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgLoadBalancer${title(var.project)}${title(var.environment)}"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_security_group" "ecs_cluster" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgEcsCluster${title(var.project)}${title(var.environment)}"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+#
+# ALB Resources
+#
+resource "aws_lb" "oar" {
+  name            = "alb${title(var.project)}${title(var.environment)}"
+  security_groups = ["${aws_security_group.alb.id}"]
+  subnets         = ["${module.vpc.public_subnet_ids}"]
+
+  tags {
+    Name        = "alb${title(var.project)}${title(var.environment)}"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_lb_target_group" "oar" {
+  name = "tg${title(var.project)}${title(var.environment)}"
+
+  health_check {
+    healthy_threshold   = "3"
+    interval            = "30"
+    matcher             = "200"
+    protocol            = "HTTP"
+    timeout             = "3"
+    path                = "/health-check/"
+    unhealthy_threshold = "2"
+  }
+
+  port     = "80"
+  protocol = "HTTP"
+  vpc_id   = "${module.vpc.id}"
+
+  target_type = "ip"
+
+  tags {
+    Name        = "tg${title(var.project)}${title(var.environment)}"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_lb_listener" "oar" {
+  load_balancer_arn = "${aws_lb.oar.id}"
+  port              = "443"
+  protocol          = "HTTPS"
+  certificate_arn   = "${module.cert.arn}"
+
+  default_action {
+    target_group_arn = "${aws_lb_target_group.oar.id}"
+    type             = "forward"
+  }
+}
+
+#
+# ECS Resources
+#
+
+resource "aws_ecs_cluster" "oar" {
+  name = "ecsCluster${title(var.project)}${title(var.environment)}"
+}
+
+data "template_file" "oar_ecs_task" {
+  template = "${file("task-definitions/oar.json")}"
+
+  vars = {
+    cpu    = "${var.fargate_cpu}"
+    image  = "${local.app_image}"
+    memory = "${var.fargate_memory}"
+
+    postgres_host     = "${module.database.hostname}"
```

Comment:
I think that this should be making use of the internal DNS record vs. the raw RDS instance FQDN.